shared/testdata.json

```diff
@@ -1101,17 +1101,17 @@
                 {
                   "query": "sum []",
                   "data": null,
-                  "expected": 0
+                  "expected": 0.0
```

Comment:
We should talk about whether sum should always return a float. If every element in the array is an int, we can probably return an int, but that makes the code more complicated.